README.md

```diff
@@ -15,14 +15,14 @@ RawCMS is a headless CMS written in ASP.NET Core, built for developers that embr
 - 👪 [Community Chat](https://gitter.im/arduosoft/RawCMS-Headless-CMS-Aspnet) - Gitter
 
 ## Contribution
-As every opensource project, we are looking for contributors. Any kind of help are welcome. *just a star to the project is a lot* If you want to contributute as developer, you can join the project  by [filling this form](https://forms.gle/dddbHWzcxypN9rpx9) or opening an issues. Any other kind of contirbution, from doc to test are also welcome.
+As every for open source project, we are looking for contributors. Any kind of help are welcome. *just a star to the project is a lot* If you want to contributute as developer, you can join the project  by [filling this form](https://forms.gle/dddbHWzcxypN9rpx9) or by opening an issue. Any other kind of contirbution, from docs to tests, are also welcome.
```

Comment:
Any kind of help are welcome. => Any kind of help is welcome.
*just a star to the project is a lot* => *Just starring the project is a lot.*
filling this form => filling out this form
 Any other kind of contirbution =>  Any other kind of contribution
are also welcome. => is also welcome